docs/source/_themes/sphinx_rtd_theme/breadcrumbs.html

```diff
@@ -0,0 +1,82 @@
+{# Support for Sphinx 1.3+ page_source_suffix, but don't break old builds. #}
```

Comment:
we have a system for docs (the devportal) and it's not this - please take all this doc generation stuff out of this repo.